modules/UcsmServer.py

```diff
@@ -19,7 +19,11 @@ def _login(self):
 
         try:
             handle.login(timeout=5)
+        except OSError as e:
+            print(e, "May be server not reachable !")
+            return
         except UcsException as e:
-            raise e
+            print(e, ", May be check for username and password !")
```

Comment:
Please change these error messages to:
`print("Problem logging in to", self.ucs_server, ":", e)`
Something similar with OSError too.

Reply:
Changed. kept the message same for both errors and "e" will describe it, like "Authentication Error " and "Url Error"

Reply:
yep, fine with me.